MapboxNavigation/CarPlayManager.swift

```diff
@@ -129,6 +128,7 @@ public class CarPlayManager: NSObject {
 
     public fileprivate(set) var mainMapTemplate: CPMapTemplate?
     public fileprivate(set) weak var currentNavigator: CarPlayNavigationViewController?
+    public static let CarPlayWaypointKey: String = "CPCarPlayWaypoint"
```

Comment:
The `CP` prefix is understood to be the CarPlay framework itself. If we have to prefix, then let’s use `MB`.